solr/CHANGES.txt

```diff
@@ -41,6 +41,11 @@ Bug Fixes
 
 * SOLR-15832: Clean-up after publish action in Schema Designer shouldn't fail if .system collection doesn't exist (Timothy Potter)
 
+Other Changes
+---------------------
+
+* SOLR-15843: Update Log4J to 2.15 (Mike Drob)
```

Comment:
I think we can classify this as a nasty securty bug, so I prefer just having one section heading in bugfix-releases.